tests/cypress/form_spec.js

```diff
@@ -0,0 +1,12 @@
+/// <reference types="cypress" />
+
+context('Navigation', () => {
+    beforeEach(() => {
+        cy.visit('http://0.0.0.0:8001')
```

Comment:
Note to me. This should change to use the base url function that cypress provides